.github/workflows/java.yml

```diff
@@ -126,7 +126,7 @@ jobs:
                     - 17
         runs-on: ubuntu-latest
         container: amazonlinux:latest
-        timeout-minutes: 15
+        timeout-minutes: 35
```

Comment:
can we make this 25?

Reply:
Our main CI is limited by 35, this one takes more time, due to VM/container deploy.
Should I update both?